src/components/SourcesList.tsx

```diff
@@ -30,20 +30,40 @@ const SourcesList = ({
                         xl: 3,
                         xxl: 3,
                     }}
-                    renderItem={(link, index) => {
+                    renderItem={(source, index) => {
                         return (
-                            <Col className="source">
-                                <List.Item key={link}>
-                                    {index + 1}.{" "}
-                                    <a
-                                        href={link}
-                                        target="_blank"
-                                        rel="noopener noreferrer"
-                                    >
-                                        {link}
-                                    </a>
-                                </List.Item>
-                            </Col>
+                            <>
+                                {typeof source === "object" ? (
+                                    <Col className="source">
+                                        <List.Item
+                                            id={source.ref}
+                                            key={source.link}
+                                        >
+                                            {index + 1}.{" "}
+                                            <a
+                                                href={source.link}
+                                                target="_blank"
+                                                rel="noopener noreferrer"
+                                            >
+                                                {source.link}
+                                            </a>
+                                        </List.Item>
+                                    </Col>
+                                ) : (
+                                    <Col className="source">
```

Comment:
todo: add a comment to remember to remove this when we do a migration for the sources schema